### ISSUE TITLE
Build failed in Android Studio 2.2.3: canOverrideExistingModule

### ISSUE THREAD

**Issue by davidkhala** [SunskyXH/DND-tools#2]
My test device Android level is 4.4
see in http://p1.bpimg.com/567571/5353d66ce731bac4.png
This is the first screen when APK launched( using Instant Run)

And also I found that should `react-native start` be before  `react-native run-android`  ?

**Comment by SunskyXH** [comment]
1.check that you had follow the steps from README
2.If you write your own NativeModule, you must override methods like
```java
 @Override    
  public boolean canOverrideExistingModule() {        
    return true;    
  }        

  @Override    
  public String getName() {        
    return "name";    
  }
```
3.`react-native run-android` will call `react-native start`,so the order does no difference.
4.I don't have Android device yet, even simulator, but I'll test this project in Android device soon

**Comment by SunskyXH** [comment]
I've query the docs, the Android SDK version should be `6.0`, and  make sure that Android SDK Build-Tools 's version is `23.0.1`.

**Comment by davidkhala** [comment]
After the typo is fixed in the commit 86628d2bd48bf91766e9ed30674264d386ba1c79, this issued is eliminated. Please close.
